OrbitCore/TracepointEventBuffer.h

```diff
@@ -37,8 +37,12 @@ class TracepointEventBuffer {
     }
   }
 
+  uint32_t GetNumTracepointsForThreadId(int32_t thread_id) const;
+
  private:
   int32_t kNotTargetProcessThreadId = -2;
```

Comment:
This should be a `static const` by the way.

Reply:
Thanks so much for pointing that out!